src/uvw/stream.hpp

```diff
@@ -155,8 +155,8 @@ class StreamHandle: public Handle<T, U> {
 
 public:
 #ifdef _WIN32
-    StreamHandle(ConstructorAccess ca, std::shared_ptr<Loop> ref)
-        : Handle{ca, std::move(ref)}
+    StreamHandle(typename Handle<T, U>::ConstructorAccess ca, std::shared_ptr<Loop> ref)
+        : Handle<T, U>{std::move(ca), std::move(ref)}
```

Comment:
I suspect that the problem is [GCC version provided with MinGW](http://www.mingw.org/wiki/GCCStatus). `uvw` requires a fully compliant C++14 compiler, GCC4.4 isn't one of them. I suspect you'll encounter more problems in future.  
In order to accept this change we should at least add GCC4.4 to Travis (see `.travis.yml`) and verify that all the tests compile and run correctly. Did you try to compile and run them under MinGW?